Tita/Controllers/LoginViewController.swift

```diff
@@ -86,6 +86,8 @@ class LoginViewController: UIViewController, UITextFieldDelegate {
 
     @objc private func onTapForgot(sender:UIButton){
         print("Forgot")
+        let nextVC = FindAccountViewController()
+        self.navigationController?.pushViewController(nextVC, animated: true)
```

Comment:
```suggestion
        let nextVC = LoginViewController()
        self.navigationController?.pushViewController(nextVC, animated: true)
```
`nextVC`는 개발이 완료됐다면 다시 바꿔주세요.